test/hpr_packet_reporter_SUITE.erl

```diff
@@ -0,0 +1,187 @@
+-module(hpr_packet_reporter_SUITE).
+
+-export([
+    all/0,
+    init_per_testcase/2,
+    end_per_testcase/2
+]).
+
+-export([
+    upload_report_test/1,
+    upload_window_test/1
+]).
+
+-include("hpr.hrl").
+-include("../src/grpc/autogen/server/packet_router_pb.hrl").
```

Comment:
I want to try to avoid using records here and instead use the `her_router` module